Packs/CortexResponseAndRemediation/Playbooks/playbook-Azure_AD_account_unlock_or_password_reset.yml

```diff
@@ -0,0 +1,1564 @@
+      id: 8716fa1c-18b7-4d93-8e5f-16ab80309eb4
+      version: -1
+      name: Evaluate Analyst Response for Next Action
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      Disable both:
+      - "36"
+      Disable resource user:
+      - "8"
+      Disable target user:
+      - "31"
+      No Action:
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: No Action
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Analyst Action.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: No Action
+          ignorecase: true
+    - label: Disable resource user
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Analyst Action.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Disable resource user
+          ignorecase: true
+    - label: Disable target user
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Analyst Action.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Disable target user
+          ignorecase: true
+    - label: Disable both
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Analyst Action.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Disable both
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1060,
+          "y": 2890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 1af19eea-5223-4d7f-8852-d18e51a9c561
+    type: regular
+    task:
+      id: 1af19eea-5223-4d7f-8852-d18e51a9c561
+      version: -1
+      name: Disable both users
+      description: |-
+        Disables a user from all Office 365 applications, and prevents sign in. Note: This command disables a user,
+        but does not terminate an existing session. Supported only in a self-deployed app flow with the
+        Permission: Directory.AccessAsUser.All(Delegated).
+      script: '|||msgraph-user-account-disable'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      user:
+        complex:
+          root: Core.OriginalAlert.raw_abioc.event.identity_orig.user
+          accessor: userPrincipalName
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: ${Core.OriginalAlert.event.raw_log.properties.targetResources.userPrincipalName}
```

Comment:
This should be from previous tasks, otherwise it will just append the DT itself
![image](https://github.com/user-attachments/assets/60ea2c84-5f06-4d86-8afd-514f9be1dd03)